docs/content/cache.md

```diff
@@ -242,6 +242,15 @@ which makes it think we're downloading the full file instead of small chunks.
 Organizing the remotes in this order yelds better results:
 <span style="color:green">**cloud remote** -> **cache** -> **crypt**</span>
 
+### Cache and Remote Control (--rc) ###
+Cache supports the new `--rc` mode in rclone and can be remote controlled through the following end points:
+
+### rc cache/expire
+Purge a remote from the cache backend. Supports either a directory or a file.
+Params:
+  - **remote** = path to remote **(required)**
+  - **withData** = true/false to delete cached data (chunks) as well _(optional)_
```

Comment:
i think it would be good to mention that **_false_** is the standard

Reply:
That makes sense. Thanks